bootcamp-milestone-2/src/app/layout.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import './globals.css'
+import Navbar from "@/app/components/navbar"
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata: Metadata = {
+  title: 'Create Next App',
+  description: 'Generated by create next app',
+}
```

Comment:
If you change this you can have the site tab say whatever you want instead of "Create Next App".